lib/format.js

```diff
@@ -0,0 +1,65 @@
+const exec = require('child-process-promise').exec;
```

Comment:
might consider to use [execa](https://github.com/sindresorhus/execa) here

Reply:
lol I saw it but assumed it was an old async combinator because of the name 😛 I'm switching over, looks nicer